src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/EntityAIRangeGuard.java

```diff
@@ -256,6 +280,12 @@ public void attackEntityWithRangedAttack(@NotNull final EntityLivingBase entityT
         worker.damageItemInHand(1);
     }
 
+    @Override
+    public void setSwingingArms(final boolean swingingArms)
+    {
+        //todo
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1135)